### PR TITLE
update massive to 5.x

### PR DESCRIPTION
**Pull request by maxcbc**
https://trello.com/c/iYjtQsKc/1313-syndication-api-massive-package-upgrade

### CODE REVIEW

File: test/server/controllers/export.spec.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const underTest = require('../../../server/controllers/export');
 
 const MODULE_ID = path.relative(`${process.cwd()}/test`, module.id) || require(path.resolve('./package.json')).name;
 
-describe(MODULE_ID, function () {
+describe.only(MODULE_ID, function () {
```
Comment (leggsimon) [inline test/server/controllers/export.spec.js]:
```suggestion
describe(MODULE_ID, function () {
```

**Comment by edds** [comment]
This PR is being closed due to inactivity as part of a Pull Request Bankruptcy. If you feel it’s still useful or relevant please feel free to re-open it.

**Comment by leggsimon** [comment]
I vaguely remember that the current version of massive had a security vulnerability, I think after fixing that test this one was ready to go

**Comment by binaryberry** [comment]
Thanks for flagging Simon, we'll merge this! 
PS: We miss youuuu 👋

**Comment by maxcbc** [comment]
I seem to think this passed tests but broke prod when i did it before

**Comment by maxcbc** [comment]
worth running up locally first

**Comment by binaryberry** [comment]
oh dear, ok will do! Do you remember *how* it broke prod by any chance @maxcbc ?

**Comment by maxcbc** [comment]
@binaryberry I can't remember exactly. However this app hooks into internal implementation details of some of the libraries it uses, so it is likely a bit of a rabbit hole to investigate.

I think if you ran it up, locally you'd probably be able to find it, write a test for it and then fix it

**Pull request merged**